### PR TITLE
DGBv0.02 == subfolder version

### LINKED ISSUE #1: Example Jupyter Notebook for TGN
Create an example Jupyter notebook which will contain the following steps
1). download dataset from dgb, say canVote
2). process dataset into TGN readable format
3). define TGN architecture 
4). train TGN on the training set while using validation set for evaluation
5). use our provided evaluator to evaluate test set performance with a). random  b). historical  c). inductive negative sampling strategies
6). report the above evaluation results

Notes: 
* don't forget to fix random seed for TGN training, should be 100% reproducible 
*  also check that dataset split is fixed and using the default split
* provide detailed instruction on dataset processing and evaluator usage

### PR DESCRIPTION
**Pull request by abubakardaud**
…with TGN, added example_TGN.ipynb

Please approve

### CODE REVIEW

File: example.py [hunk 1]
```diff
@@ -6,14 +6,14 @@
     '''
     #! step 1: download dataset from url, select specific dataset using str name 
     '''
+    print("Running main for DGB")
     input_list = "canparl" #"enron"
-    enron_tgb = TemporalDataSets(data_name=input_list)
+    enron_tgb = TemporalDataSets(data_name=input_list, sub_folder_name="hello_kity")
```
Comment (shenyangHuang) [inline example.py:11]: might need a better name like "tg_rocks"

**Review by shenyangHuang — CHANGES_REQUESTED** [review]
update the name :)
